api/src/services/generate-xform.js

```diff
@@ -16,17 +16,28 @@ const MEDIA_SRC_ATTR = ' data-media-src="';
 
 const FORM_STYLESHEET = path.join(__dirname, '../xsl/openrosa2html5form.xsl');
 const MODEL_STYLESHEET = path.join(__dirname, '../../node_modules/enketo-xslt/xsl/openrosa2xmlmodel.xsl');
+const XSLTPROC_CMD = 'xsltproc';
 
 const transform = (formXml, stylesheet) => {
   return new Promise((resolve, reject) => {
-    const xsltproc = childProcess.spawn('xsltproc', [ stylesheet, '-' ]);
+    const xsltproc = childProcess.spawn(XSLTPROC_CMD, [ stylesheet, '-' ]);
     let stdout = '';
     let stderr = '';
     xsltproc.stdout.on('data', data => stdout += data);
     xsltproc.stderr.on('data', data => stderr += data);
     xsltproc.stdin.setEncoding('utf-8');
-    xsltproc.stdin.write(formXml);
-    xsltproc.stdin.end();
+    try {
+      xsltproc.stdin.write(formXml);
+    } catch (err) {
+      if (err.code === 'EPIPE') {
+        return reject(
+          new Error(`Unable to continue execution, check that '${XSLTPROC_CMD}' command is available.`));
```

Comment:
Since we're already going 2 lines, we can also do this, instead of breaking in there.
```js
const errorMsg = `Unable to continue execution, check that '${XSLTPROC_CMD}' command is available.`;
return reject(new Error(errorMsg));
```

Reply:
or if that's still exceeding the char limit then close the first bracket in separate line

```
return reject(
  new Error(`Unable to continue execution, check that '${XSLTPROC_CMD}' command is available.`)
); // close here
```